app/src/main/java/org/dhis2/usescases/login/LoginPresenter.kt

```diff
@@ -2,30 +2,33 @@ package org.dhis2.usescases.login
 
 import android.os.Build
 import co.infinum.goldfinger.Goldfinger
-import co.infinum.goldfinger.rx.RxGoldfinger
 import io.reactivex.Observable
 import io.reactivex.disposables.CompositeDisposable
 import org.dhis2.App
 import org.dhis2.R
 import org.dhis2.data.fingerprint.FingerPrintController
 import org.dhis2.data.fingerprint.Type
 import org.dhis2.data.prefs.Preference
+import org.dhis2.data.prefs.Preference.Companion.PIN
+import org.dhis2.data.prefs.Preference.Companion.SESSION_LOCKED
 import org.dhis2.data.prefs.PreferenceProvider
 import org.dhis2.data.schedulers.SchedulerProvider
 import org.dhis2.data.server.UserManager
 import org.dhis2.usescases.main.MainActivity
 import org.dhis2.utils.Constants.*
-import org.dhis2.utils.analytics.*
 import org.dhis2.utils.analytics.ACCOUNT_RECOVERY
-import org.hisp.dhis.android.core.d2manager.D2Manager
+import org.dhis2.utils.analytics.AnalyticsHelper
+import org.dhis2.utils.analytics.CLICK
+import org.dhis2.utils.analytics.LOGIN
+import org.dhis2.utils.analytics.SERVER_QR_SCANNER
 import org.hisp.dhis.android.core.maintenance.D2Error
 import org.hisp.dhis.android.core.maintenance.D2ErrorCode
 import org.hisp.dhis.android.core.systeminfo.SystemInfo
 import retrofit2.Response
 import timber.log.Timber
 
 class LoginPresenter(private val view: LoginContracts.View,
```

Comment:
We need to also update the LoginPresenter Tests with the new name of the SharedPreferences' Constants names.

Reply:
Next time the build will fail :)